indico/modules/events/tracks/templates/display.html

```diff
@@ -21,26 +21,52 @@
     {% endif %}
 
     <ul class="tracks flexcol f-j-space-between">
-        {% for track in tracks %}
-            <li>
-                <div class="track-title flexrow f-a-center">
-                    <h3 class="f-self-stretch">{{ track.title }}</h3>
-                    {% if track.code %}
-                        <span class="track-code">
-                            {{- track.code -}}
-                        </span>
-                    {%- endif %}
-                    {% if track.can_convene(session.user) or track.can_review_abstracts(session.user) %}
-                        <a href="{{ url_for('abstracts.display_reviewable_track_abstracts', track) }}"
-                           class="i-link icon-stack-text track-link"
-                           title="{% trans %}Review abstracts in this track{% endtrans %}">
-                        </a>
-                    {% endif %}
-                </div>
-                <div class="description">
-                    {{- track.description -}}
-                </div>
-            </li>
+        {% for item in items %}
+            {% if item.__classname__ == 'tracks' %}
```

Comment:
where is `__classname__` coming from? in any case, this doesn't look like a good idea... i'd rather add a `is_track_group = True` attribute to `TrackGroup` (and with `False` to `Track`) and check that...

Reply:
It was supposed to be `__tablename__` but I'm guessing the same applies for that too.